src/client/delphi_epidata.py

```diff
@@ -665,3 +665,44 @@ def covid_hosp_facility_lookup(
       raise Exception('one of `state`, `ccn`, `city`, `zip`, or `fips_code` is required')
     # Make the API call
     return Epidata._request(params)
+
+  # Fetch Delphi's COVID-19 Surveillance Streams
+  @staticmethod
+  def covidcast_nowcast(
+          data_source, signals, time_type, geo_type,
+          time_values, geo_value, as_of=None, issues=None, lag=None, **kwargs):
+    """Fetch Delphi's COVID-19 Nowcast sensors"""
+    # also support old parameter name
+    if signals is None and 'signal' in kwargs:
+      signals=kwargs['signal']
```

Comment:
This is a new endpoint so there is no need to make it backwards-compatible with anything
```suggestion
```